server/handler/export.go

```diff
@@ -0,0 +1,47 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"k8s.io/klog/v2"
+
+	"github.com/kubernetes-sigs/kube-scheduler-simulator/export"
+	"github.com/kubernetes-sigs/kube-scheduler-simulator/server/di"
+)
+
+type ExportHandler struct {
+	service di.ResourcesService
+}
+
+func NewExportHandler(s di.ResourcesService) *ExportHandler {
+	return &ExportHandler{service: s}
+}
+
+func (h *ExportHandler) Export(c echo.Context) error {
+	ctx := c.Request().Context()
+
+	rs, err := h.service.Export(ctx)
+	if err != nil {
+		klog.Errorf("failed to export all resources: %+v", err)
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+	return c.JSON(http.StatusOK, rs)
+}
+
+func (h *ExportHandler) Import(c echo.Context) error {
+	ctx := c.Request().Context()
+
+	reqResources := new(export.ResourcesApplyConfiguration)
```

Comment:
Please define the same struct as `export.ResourcesApplyConfiguration` in this file, use it to bind requests and convert it into `export.ResourcesApplyConfiguration` to pass struct to service.

Generally, we should not make a handler layer dependent on a service layer, because changing any service layer will cause breaking changes on handler layer. 
In this case, if we change `export.ResourcesApplyConfiguration` in the future, `/import` API will be changed.

Reply:
So,  in calling of function will be like this?
```
	err := h.service.Import(ctx, &export.ResourcesApplyConfiguration{
		Pods:            reqResources.Pods,
		Nodes:           reqResources.Nodes,
		Pvs:             reqResources.Pvs,
		Pvcs:            reqResources.Pvcs,
		StorageClasses:  reqResources.StorageClasses,
		SchedulerConfig: reqResources.SchedulerConfig,
	})
```

Reply:
Yes. (Or, create new function to convert new struct to `export.ResourcesApplyConfiguration`)